### PR TITLE
Add receptorctl connect doc

### PR DESCRIPTION
**Pull request by thom-at-redhat**

### CODE REVIEW

File: docs/source/receptorctl/receptorctl_connect.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+-------
+connect
+-------
+
+.. contents::
+   :local:
+
+``receptorctl connect`` connects the local client to the Receptor connected node.
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_connect.rst]:
```suggestion
``receptorctl connect`` establishes a connection between local client and a Receptor node.
```

File: docs/source/receptorctl/receptorctl_connect.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+.. code-block:: text
+
+  ss --listening --processes --unix 'src = unix:<socket_path>'
+  Netid         State          Recv-Q         Send-Q                   Local Address:Port                     Peer Address:Port        Process
+  u_str         LISTEN         0              4096                   /tmp/local.sock 38130170                            * 0            users:(("receptor",pid=3226769,fd=7))
+
+``ps -fp $(pidof receptor)``
+``lsof -p <pid>``
+
+``remote_node`` is the node identifier for the Receptor connected node.
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_connect.rst]:
```suggestion
``remote_node`` is the identifier of a Receptor node.
```

File: docs/source/receptorctl/receptorctl_connect.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+``remote_control_service`` is the service name of the Receptor connected node.
```
Comment (oraNod) [inline docs/source/receptorctl/receptorctl_connect.rst]:
```suggestion
``remote_control_service`` is the service name of a Receptor node.
```

File: docs/source/conf.py [hunk 4]
```diff
@@ -66,49 +62,63 @@
...
 # -- Options for Texinfo output -------------------------------------------
 
 # Grouping the document tree into Texinfo files. List of tuples
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-    (master_doc, 'receptor', 'receptor Documentation',
-     author, 'receptor', 'Overlay network to establish a persistent mesh.',
-     'Miscellaneous'),
+    (master_doc, "receptor", "receptor Documentation", author, "receptor", "Overlay network to establish a persistent mesh.", "Miscellaneous"),
```
Comment (oraNod) [inline docs/source/conf.py]:
```suggestion
    (
        master_doc,
        "receptor",
        "receptor Documentation",
        author,
        "receptor",
        "Overlay network to establish a persistent mesh.",
        "Miscellaneous",
    ),
```

Comment (oraNod) [reply docs/source/conf.py]: Just a nit but I'd prefer an expanded format like this for better readability.

File: docs/source/conf.py [hunk 5]
```diff
@@ -31,29 +27,29 @@
...
+pygments_style = "ansible"
+language = "en"
+master_doc = "index"
+source_suffix = ".rst"
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_ansible_theme'
+html_theme = "sphinx_ansible_theme"
```
Comment (oraNod) [inline docs/source/conf.py]: Looks like this is missing from `docs/requirements.txt`

File: docs/source/conf.py (hunk 4 above)
Comment (thom-at-redhat) [reply docs/source/conf.py]: I tried but black puts it back :-/

Comment (oraNod) [reply docs/source/conf.py]: @thom-at-redhat hmm. which version of black are you using?

When I format with black it actually gives me the expanded list. I wonder if I'm doing something weird.

```
$ black --version
black, 24.4.2 (compiled: yes)
Python (CPython) 3.12.3

$ black docs/source/conf.py

$ grep texinfo_documents -A 9 docs/source/conf.py
texinfo_documents = [
    (
        master_doc,
        "receptor",
        "receptor Documentation",
        author,
        "receptor",
        "Overlay network to establish a persistent mesh.",
        "Miscellaneous",
    ),
```

**Review by oraNod — APPROVED** [review]

File: docs/source/conf.py (hunk 4 above)
Comment (thom-at-redhat) [reply docs/source/conf.py]: you are correct:
```
$ black --version
black, 24.4.2 (compiled: yes)
Python (CPython) 3.12.3
```
It reformatted correctly the second time, thank you for your help!

**Review by matoval — APPROVED** [review]

**Pull request merged**